src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -949,6 +949,118 @@ julia> volume(p)
 gelfand_tsetlin_polytope(lambda::AbstractVector) = Polyhedron{QQFieldElem}(Polymake.polytope.gelfand_tsetlin(Polymake.Vector{Polymake.Rational}(lambda), projected = false))
 
 @doc raw"""
+    demazure_character(lambda::AbstractVector, sigma::PermGroupElem)
+
+Construct  the Demazure character indexed by a weakly decreasing vector `lambda` and a permutation `sigma`.
+
+# Examples
+```jldoctest
+julia> lambda = Partition([3,1,1])
+[3, 1, 1]
+
+julia> w0 = @perm (1,3,2)
+(1,3,2)
+
+julia> dc = demazure_character(lambda,w0)
+x1^3*x2*x3 + x1^2*x2^2*x3 + x1*x2^3*x3
+```
+"""
+function demazure_character(lambda::AbstractVector, sigma::PermGroupElem)
+       genGT = gelfand_tsetlin_polytope(lambda,sigma)
+       R = polynomial_ring(ZZ, length(lambda))
+       x = R[2]
+       s = R[1](0)
+       for P in lattice_points(genGT)
+          w=[sum([P[binomial(n+1,2)-binomial(i+1,2)+j] for j in 1:i]) for i in 1:length(lambda)]
+          s=s+x[1]^w[1]*prod([x[i]^(w[i]-w[i-1]) for i in 2:n])
+       end
+       return s
+end
```

Comment:
```suggestion
function demazure_character(lambda::AbstractVector, sigma::PermGroupElem)
  genGT = gelfand_tsetlin_polytope(lambda,sigma)
  R = polynomial_ring(ZZ, length(lambda))
  x = R[2]
  s = R[1](0)
  for P in lattice_points(genGT)
    w=[sum([P[binomial(n+1,2)-binomial(i+1,2)+j] for j in 1:i]) for i in 1:length(lambda)]
    s=s+x[1]^w[1]*prod([x[i]^(w[i]-w[i-1]) for i in 2:n])
  end
  return s
end
```
We have 2-space indent in the editorconfig, please check if you can use that in your editor (https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Editor-configuration.